readme.md

```diff
@@ -504,10 +504,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 
 ## Other Awesome Lists
 
-- [awesome-git-addons](https://github.com/stevemao/awesome-git-addons) - A curated list of addons that extends git cli.
-- [awesome-macos-command-line](https://github.com/herrbischoff/awesome-macos-command-line) - A curated list of shell commands and tools specific to macOS.
-- [awesome-command-line-apps](https://github.com/herrbischoff/awesome-command-line-apps) – An alternative awesome list of cli apps.
-- [awesome-shell](https://github.com/alebcay/awesome-shell) – A curated list of awesome command-line utilities and frameworks.
+- [awesome-git-addons](https://github.com/stevemao/awesome-git-addons#readme) - A curated list of addons that extends git cli.
```

Comment:
Could you order them in relevancy?

Reply:
How so? Want me to bump git-addons to the bottom?

Reply:
Just order them by importance (in relation to this list)
e.g. command-line-apps > macos-clis > git-addons